python/cuml/dask/metrics/confusion_matrix.py

```diff
@@ -0,0 +1,117 @@
+#
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+import cupy as cp
+from cuml.dask.common.input_utils import DistributedDataHandler
+
+from cuml.dask.common.utils import get_client
+
+from cuml.utils.memory_utils import with_cupy_rmm
+from cuml.dask.metrics.utils import sorted_unique_labels
+from cuml.prims.label import make_monotonic
+
+
+@with_cupy_rmm
+def _local_cm(inputs, labels, use_sample_weight):
+    if use_sample_weight:
+        y_true, y_pred, sample_weight = inputs
+    else:
+        y_true, y_pred = inputs
+        sample_weight = cp.ones(y_true.shape[0], dtype=y_true.dtype)
```

Comment:
I believe it will be better to use `cumlarray.ones` in place of `cp.ones`

Reply:
See other comment.